### ISSUE TITLE
missing symplify for MatrixSymbol with neg 

### ISSUE THREAD

**Issue by vruge** [sympy/sympy#13490]
following example failed to simplify `S.NegativeOne * S.NegativeOne`
```python
n, m = symbols('n m')
A = MatrixSymbol('A', n, m)
assert(A == = simplify(-(-A)) )
```

**Comment by oscarbenjamin** [comment]
Your example has a syntax error because of the extra `=`. Deleting that it seems to work fine:
```julia
In [3]: n, m = symbols('n m') 
   ...: A = MatrixSymbol('A', n, m) 
   ...: assert(A == simplify(-(-A)) 
```
(No AssertionErrror)

**Comment by vruge** [comment]
@oscarbenjamin Which version of sympy your using? 
If I run
```bash
import sympy
import sys

print('python-version', sys.version) 
print('sympy:', sympy.__version__) 

n, m = sympy.symbols('n m')
A = sympy.MatrixSymbol('A', n, m)
print(A) 
print(sympy.simplify(-(-A))) # (-1)*(-1)*A
assert(A == sympy.simplify(-(-A)))
```

than I get
```
('python-version', '2.7.15rc1 (default, Nov 12 2018, 14:31:15) \n[GCC 7.3.0]')
('sympy:', '1.3')
A
(-1)*(-1)*A
Traceback (most recent call last):
  File "test.py", line 12, in <module>
    assert(A == sympy.simplify(-(-A)))
AssertionError
```

**Comment by oscarbenjamin** [comment]
I see the same error as you in version 1.3. However this works fine in master so you should see it fixed in 1.4.
